fractal-puppy/Engine
# Persist player data between levels

In this pr the player progress (items, skills, level, xp, stats) will remain if the player change the level.

To test this:
- Recompile everything
- Load the scene Level0-TheGraveyard and play.
- Unlock some skills, grab some items from the map and equip, kill a cuple of enemies and equip skills/items in the 1,2,3,4,q,w,e,r buttons of the hud.
- Then finish the level (arrive to the final bridge of the level) and then the credits jump.
- Then stop the game and load the graveyard scene again. You have to see the same skills unlocked, items grabbed and equiped, xp, level and stats.

NOTE: If you click in new game in the main menu, deletes all saved information and starts from 0.
